tests/e2e-playwright/helpers/api/contexts.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-console */
+import { APIRequestContext, request } from '@playwright/test';
+import { retry } from '@lifeomic/attempt';
+import { API } from '@e2e/factories/users';
+import { getContextByRole } from '../context';
+
+const NONCE_PATH = '/wp-admin/admin-ajax.php?action=rest-nonce';
+
+type RequestRunner = ( WpApiRequestContext ) => void;
+
+/**
+ * Run callback over a separated temporary context
+ * avoiding to mess the test context.
+ *
+ * @param callback A callback function to run requests using the admin context
+ * @return Promise<void>
+ */
+export const asAdmin = async ( callback: RequestRunner ): Promise< void > => {
+	const context = await request.newContext( {
+		baseURL: 'http://localhost:8889',
```

Comment:
to make sure, isn't this address configurable?

Reply:
It is, we can improve it later, to focus now if just kill the get nonce issue.